kobo/apps/audit_log/views.py

```diff
@@ -174,16 +175,40 @@ class AuditLogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
     ]
 
 
-@extend_schema(tags=['access-logs'])
+@extend_schema(
+    tags=['Access-Logs'],
+    description=read_md('audit_log', 'access_logs/list'),
+    responses=open_api_200_ok_response(AccessLogListInlineSerializer),
+)
 class AllAccessLogViewSet(AuditLogViewSet):
+    """
+    ViewSet for managing superusers' access logs. Only available to superusers.
```

Comment:
```suggestion
    ViewSet for managing all users' access logs. Only available to superusers.
```
this endpoint is not just for superuser's logs. it's for everyone's logs, it's just only available to superusers